binary_transparency/firmware/cmd/flash_tool/impl/flash_tool.go

```diff
@@ -286,8 +282,12 @@ func verifyAnnotations(ctx context.Context, c *client.ReadonlyClient, pb api.Pro
 		return fmt.Errorf("inclusion proof calculated root %x but wanted %x", calc, mcp.RootHash)
 	}
 
+	var agg api.AggregatedFirmware
+	if err := json.NewDecoder(bytes.NewReader(preimage)).Decode(&agg); err != nil {
```

Comment:
Can do ` ... := json.Unmarshal(preimage, &agg); ...`

Reply:
I looked for this under json.Decode and then wrote it off as a figment of my imagination that such an obvious thing would be possible. Thanks for the pointer!